### PR TITLE
load filters if available

### PR DESCRIPTION
**Pull request by gravitypriest**
Signed-off-by: Jeremy Crafts <jcrafts@redhat.com>

### CODE REVIEW

**Comment by gravitypriest** [comment]
Core Collection Part 1

**Comment by gravitypriest** [comment]
A thought: should we panic if called from the client and filters are missing?

**Comment by gravitypriest** [comment]
In addition, fall back to `hostname` if `hostname -f` fails

File: insights/collect.py [hunk 1]
```diff
@@ -246,7 +246,21 @@ def collect(manifest=default_manifest, tmp_path=None, compress=False):
 
     to_persist = get_to_persist(client.get("persist", set()))
 
-    hostname = call("hostname -f", env=SAFE_ENV).strip()
+    try:
+        filters.load()
+    except IOError as e:
+        if e.errno == 0:
```
Comment (bfahr) [inline insights/collect.py:252]: I get errno = 2 when no file is available.

**Pull request merged**